src/components/SignUp/PersonSignupFlow.tsx

```diff
@@ -0,0 +1,323 @@
+import React, { Component, ReactComponentElement } from 'react';
+import { Redirect } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { withAlert } from 'react-alert';
+import { Steps } from 'antd';
+import { ProgressBar } from 'react-bootstrap';
+import getServerURL from '../../serverOverride';
+// import Logo from '../../static/images/logo.svg';
+import 'antd/dist/antd.css'; // or 'antd/dist/antd.less'
+import AccountSetup from './AccountSetup';
+import PersonalInformation from './PersonalInformation';
+import SignUserAgreement from './SignUserAgreement';
+import ReviewSubmitInviteSignupVersion from './ReviewSubmitInviteSignupVersion';
+import Role from '../../static/Role';
+import ReviewSubmit from './ReviewSubmit';
+
+const { Step } = Steps;
+
+interface Props {
+  alert: any,
+  personRole: Role,
+  //invited: boolean
+  //orgName: string,for generalizing to invite
+}
+interface State {
+  signupStage: number,
+  username: string,
+  password: string,
+  confirmPassword: string,
+  firstname: string,
+  lastname: string,
+  birthDate: Date,
+  email: string,
+  phonenumber: string,
+  address: string,
+  city: string,
+  state: string,
+  zipcode: string,
+  hasSigned: boolean,
+  recaptchaPayload: string,
+  buttonState: string,
+  redirectLogin: boolean,
+}
+
+class PersonSignupFlow extends Component<Props, State, {}> {
```

Comment:
Actually can you import this function from the CompleteSignupFlow file? Just so we dont have duplicate code lying around